DESCRIPTION

```diff
@@ -1,15 +1,52 @@
 Package: geoGraph
-Version: 1.0-0
-Date: 2015/09/13
-Title: walking through the geographic space using graphs
-Author: Thibaut Jombart
+Type: Package
+Title: What the Package Does (Title Case)
+Version: 1.0-1
```

Comment:
I have moved to semantic since, so would personally use 1.1.1.

Reply:
Agreed